.github/workflows/R-CMD-check.yaml

```diff
@@ -116,6 +116,7 @@ jobs:
       # worth testing
       - name: Install testing dependencies
         run: |
+          remotes::install_version("showtext", "0.8-1")
```

Comment:
For reference https://github.com/yixuan/showtext/issues/43

I'm not sure if this is a problem for `svg()`, so another solution could be to use that for the visual testing